modules/gapi/src/backends/common/serialization.cpp

```diff
@@ -266,6 +266,19 @@ I::IStream& operator>> (I::IStream& is,       cv::GOpaqueDesc &) {return is;}
 I::OStream& operator<< (I::OStream& os, const cv::GArrayDesc &) {return os;}
 I::IStream& operator>> (I::IStream& is,       cv::GArrayDesc &) {return is;}
 
+#if !defined(GAPI_STANDALONE)
+I::OStream& operator<< (I::OStream& os, const cv::UMat &)
+{
+    GAPI_Assert("Serialization: Unsupported << for UMat");
```

Comment:
This assert won't work since the string casts to `true`.

This one will:

```c++
GAPI_Assert(false && "Oops");
```

Reply:
Done